l10n_eu_product_adr/scripts/import_adr_multilang_xlsx.py

```diff
@@ -176,12 +176,10 @@ def parse_limited_quantity(row, split=True):
         if "BP 251" in value:  # known case
             return False, False
         else:
-            raise ValueError("Cannot parse limited quantity: %s (%s)" % (value, row))
+            raise ValueError(f"Cannot parse limited quantity: {value} ({row})")
```

Comment:
This should be translatable (so no f-string).

Reply:
Yeah, but this is not a migration problem, but a bug detected reviewing this. This string should be translatable (so no f-string and put `_(`)